CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+* Remove usage of goog.isFunction.
+* Fix FirebaseUI-web spinner off for RTL languages.
+* Show warning on accountchooser.com becomes universally opt-out.
+* Add unit test to confirm that if accountchooser.com JS dependency fails to load that the sign in flow would continue to work as expected.
```

Comment:
We can remove this one (unit test). We just need to focus on external facing changes.